Src/BootCamp.Chapter/Program.cs

```diff
@@ -7,6 +7,55 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello World!");
+
+            //Get Name
+            Console.Write("What is your name? ");
+            var myName = Console.ReadLine();
+            Console.WriteLine("Hello, " + myName + "!");
+            //Get Age
+            Console.Write("How old are you? ");
+            var myAge = Console.ReadLine();
```

Comment:
Age should be converted to int :) It's not very significant in this case, because it's just used for display, but it's a more correct data type for storing age- whole numbers and not text.